CHANGELOG.rst

```diff
@@ -1,5 +1,27 @@
 .. _changelog:
 
+0.7.3
+-----
+
+This is a minor release that contains mainly improvements and fixes to the Renku CLI.
+Templates now use this new Renku CLI version ``0.12.2`` by default.
+
+New features
+~~~~~~~~~~~~
+
+- **Migrations** A ``migrationscheck`` command has been added.
+- **Git LFS** Large files are automatically added to LFS.
```

Comment:
This has actually been the case for some time - I would maybe rephrase this with "When a session is closed and work is automatically saved, large files are added to LFS according to the project's settings (see the [`lfs_threshold` configuration](https://renku-python.readthedocs.io/en/v0.12.0/commands.html#available-configuration-values))"

Reply:
This makes more sense, I could not get that from the release notes. Thanks for the clarification!

Reply:
well technically we were asking for confirmation before and now it can be configured to just happen automatically. But I'm not sure a user will necessarily make that distinction

Reply:
argh I linked to the wrong docs - should be https://renku-python.readthedocs.io/en/v0.12.2/commands.html#available-configuration-values